### ISSUE TITLE
Strings not embedded properly in Parallel Coordinate plot.

### ISSUE THREAD

**Issue by cemde** [optuna/optuna-dashboard#198]
### Description

TL;DR discrete hyperparameters with numbers in the string are not represented correctly in the parallel coordinate plots.
 
I have a hyperparameter `configuration` which regulates the configuration of a MLP which looks like this "-1:512:64:512:-1", because the `suggest_categorical` does not support tuples or lists.

In the Parallel coordinate plot, this string is represented by `int: -1` and thus all configurations are mapped to the same y coordinate.

### How to Reproduce

Start an optimization with a hyper-parameter:
`suggest_categorical('hp1', ["-1:512:64:512:-1",  "-1:512:16:512:-1"])`

The parallel coordinate plot will show both configurations as 1 point.

This problem can be circumvented by using ["A:-1:512:64:512:-1",  "B:-1:512:16:512:-1"]

### Python version

3.9

### Optuna version

optuna 2.10.0

### optuna-dashboard version or git revision

optuna-dashboard 0.6.1

### Web browser

Firefox
